neard/Cargo.toml

```diff
@@ -30,6 +30,7 @@ tikv-jemallocator = { version = "0.4.0", optional = true }
 nearcore = { path = "../nearcore" }
 near-primitives = { path = "../core/primitives" }
 near-performance-metrics = { path = "../utils/near-performance-metrics" }
+near-state-viewer = { path = "../test-utils/state-viewer", package = "state-viewer" }
```

Comment:
I wonder if, as a seprate step, we should move the code itelf somewhere? `test-utils` is just an odd thing to depend on from prod. It's also OK as is -- I don't think our directory organization is super-great anyway.

Reply:
`state-viewer` isn't a test util by util, moving it somewhere sounds good, let me follow up.